### PR TITLE
0.1

### PR DESCRIPTION
**Pull request by cblanquera**

### CODE REVIEW

File: admin/AdminPackage.php [hunk 1]
```diff
@@ -98,13 +98,20 @@ public function error(
...
+    }
+
     //get the path
     $path = $request->getPath('string');
     //if not an admin path
     if ($path !== '/admin' && strpos($path, '/admin/') !== 0) {
       return;
     }
 
+    $debug = strpos($path, '/admin/spa/') === false ? 'page': 'spa';
```
Comment (lightshire) [inline admin/AdminPackage.php:113]: should we make this into constants, or is this okay since something we won't need to look back at?

Comment (cblanquera) [reply admin/AdminPackage.php:113]: oh yea your right! nice catch. Totally forgot 👍

**Review by lightshire — APPROVED** [review]

File: admin/AdminPackage.php (hunk 1 above)
Comment (cblanquera) [reply admin/AdminPackage.php:113]: ill make a note na lang to go through the packages and declare things that should be constant.

Comment (lightshire) [reply admin/AdminPackage.php:113]: i'll do the same if ever i run to some. boy scout mode haha

**Pull request merged**